dictionaries/condensed-strokes.json

```diff
@@ -548,11 +578,19 @@
 "KOL/HREBGT/-BL": "collectable",
 "KOL/HREBGT/EUBL": "collectible",
 "KOL/TP-PL": "col.",
+"KOPB/SAOEL/*PLT": "concealment",
```

Comment:
@paulfioravanti I don't have a good plan at the moment for how to handle `"KOPB/"` entries but I just wanted to note that `"KAUPB/"` is the preferred prefix for `{con^}`. I think in an ideal world we'd remove all the KOPB/ entries as misstrokes, but it gets tricky to navigate with entries like `"KOPB/K-L": "conical",` that are phonetic rather than prefixed.